saveSvgAsPng.js

```diff
@@ -151,8 +272,6 @@
         clone.setAttribute('transform', clone.getAttribute('transform').replace(/translate\(.*?\)/, ''));
 
         var svg = document.createElementNS('http://www.w3.org/2000/svg','svg')
-        svg.appendChild(clone)
```

Comment:
I guess I pressed a wrong button in vim and the code was removed in mistake :).

Added it back.